### PR TITLE
Adding upstream register and remove NSE

### PR DESCRIPTION
**Pull request by fkautz**

### CODE REVIEW

File: controlplane/cmd/nsmd/nsmd.go [hunk 1]
```diff
@@ -14,11 +17,23 @@ import (
 func main() {
 	model := model.NewModel()
 
+	registryAddress := os.Getenv("K8S_REGISTRY_ADDRESS")
```
Comment (fkautz) [inline controlplane/cmd/nsmd/nsmd.go]: On second look, I don't like this name. It is K8s specific when it should be NSM specific.

File: controlplane/cmd/nsmd/nsmd.go [hunk 2]
```diff
@@ -14,11 +17,23 @@ import (
...
+	registryAddress = strings.TrimSpace(registryAddress)
+	if registryAddress == "" {
+		registryAddress = "localhost:5000"
+	}
+
+	registryConn, err := grpc.Dial("localhost:5000")
```
Comment (edwarnicke) [inline controlplane/cmd/nsmd/nsmd.go]: Shouldn’t this be registryAddress rather than a literal?

Comment (fkautz) [inline controlplane/cmd/nsmd/nsmd.go]: Going to add a defer close here

Comment (fkautz) [reply controlplane/cmd/nsmd/nsmd.go]: good catch

**Pull request merged**